app/src/views/OperationalLearnings/i18n.json

```
@@ -0,0 +1,7 @@
+{
+    "namespace": "common",
```

When we create a new page. We should set the namespace for strings to that page. This is done so that all the strings for the particular namespace are fetched dynamically when the page is loaded.
```suggestion
    "namespace": "operational-learnings",
```